webapp/components/login/login_controller.jsx

```diff
@@ -209,7 +209,7 @@ export default class LoginController extends React.Component {
     }
 
     finishSignin(team) {
-        GlobalActions.emitInitialLoad(
+        loadMe(
```

Comment:
maybe im not reading this correctly but it this is called after `weblogin` that in turns calls the redux action `login` then `loadMe` is not necessary as you are going to perform the same requests more than once